symengine/logic.cpp

```diff
@@ -123,6 +123,27 @@ RCP<const Boolean> contains(const RCP<const Basic> &expr,
     }
 }
 
+RCP<const Basic> piecewise(const PiecewiseVec &vec)
+{
+    PiecewiseVec new_vec;
+    for (auto &p: vec) {
+        if (eq(*p.second, *boolFalse)) {
+            continue;
+        } else if (eq(*p.second, *boolTrue)) {
+            new_vec.push_back(p);
+            break;
```

Comment:
What happens if multiple cases are true at the same time? `Piecewise((1, t<0), (1, t<0))` or worse `Piecewise((1, t<0), (2, t<0))`. Sympy does this:

```python
sympy.Piecewise((1, t<0), (2, t<0))
Out[1]: Piecewise((1, t < 0))
```